app/shared/availability-view/availability-view.less

```diff
@@ -30,6 +30,23 @@
       cursor: pointer;
     }
   }
+
+  .date-picker {
+    border: none;
+    color: black;
+    cursor: pointer;
+    input {
+      background: none;
+      color: white;
+      cursor: pointer;
+    }
+    .react-date-field__calendar-icon {
```

Comment:
- [ ] The calendar icon could be removed as you suggested. Maybe later we can improve this and remove the `DateField` from here altogether but for now just removing the icon is good.